package.json

```diff
@@ -21,7 +21,7 @@
     "hubot-scripts": "^2.17.2",
     "hubot-slack": "^4.5.5",
     "hubot-spark": "git+https://github.com/tonybaloney/hubot-spark.git#c440504",
-    "hubot-stackstorm": "^0.11.3",
+    "hubot-stackstorm": "^0.12.0",
```

Comment:
You need to also update the [npm-shrinkwrap.json](https://github.com/StackStorm/st2chatops/blob/master/npm-shrinkwrap.json) lock file so the new version will be really installed.

Reply:
The documentation on how to do that properly is here: https://github.com/StackStorm/st2chatops/issues/104
Pinned the issue again as it's not the first time it's missed.

Reply:
done. Hopefully it works.